### PR TITLE
Fix quick-submit form styling

### PR DESCRIPTION
**Pull request by andreadelrio**
This is to cover issue #88

### CODE REVIEW

File: src/components/ItemForm.js [hunk 1]
```diff
@@ -23,58 +23,54 @@ const renderGeoField = ({ input, label, type, meta: { touched, error } }) => {
...
           </FormGroup>
           <FormGroup>
             <Label>Summary</Label>
-            <Field name='summary' className='title' placeholder='Summary placeholder' component={renderTextField} type='text'/>
+            <Field name='summary' className='title' placeholder='Placeholder for summary ' component={renderTextField} type='text'/>
           </FormGroup>
           <Row className='imgField'>
-            <Col xs='2' className='pr-0'>
-              <img src={imgIcon} alt=""/>
+            <Col xs={2} sm={1}>
```
Comment (dethe) [inline src/components/ItemForm.js]: Are `xs` and `sm` hints for breakpoints?

**Comment by dethe** [comment]
@andreadelrio Is this ready for review?

File: src/components/ItemForm.js (hunk 1 above)
Comment (andreadelrio) [inline src/components/ItemForm.js]: @dethe Yes, they are part of Bootstrap's responsive utilities.

**Comment by andreadelrio** [comment]
@dethe yes, this PR is ready for review.

File: src/components/ItemForm.js (hunk 1 above)
Comment (dethe) [inline src/components/ItemForm.js]: Ah, gotcha. My last project was using an old version of Bootstrap, they looked kind of familiar, but not quite. Thanks!

**Pull request merged**